.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,35 @@
+name: Codespell
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
```

Comment:
Shouldn't the codespell action show up on this PR? I don't see it

Reply:
running